composer.json

```diff
@@ -48,7 +48,7 @@
 		"keboola/phpunit-retry-annotations": "^0.5",
 		"keboola/retry": "^0.5.0",
 		"keboola/table-backend-utils": ">=2.9.0",
-		"phpcompatibility/php-compatibility": "*",
+		"phpcompatibility/php-compatibility": "dev-develop",
```

Comment:
Nevim k cemu tenhle plugin je, ale posledni verze byla vydana 5 let zpatky :slightly_smiling_face: a zaclo to hazet false positive errory, jak jsem pridal metody na enumy. Switch na dev verzi to vyresil, ale asi neni idelani to takle mit forever.